examples/scene/face_picking.py

```diff
@@ -82,30 +83,42 @@ def on_transform_change(event):
 shading_filter.shading = next(shading)
 
 
+throttle = time.monotonic()
+
+
 @canvas.events.mouse_move.connect
 def on_mouse_move(event):
+    global throttle
+    # throttle mouse events to 50ms
+    if time.monotonic() - throttle < 0.05:
+        return
+    throttle = time.monotonic()
+
+    # adjust the event position for hidpi screens
+    render_size = tuple(d * canvas.pixel_scale for d in canvas.size)
+    x_pos = event.pos[0] * canvas.pixel_scale
+    y_pos = render_size[1] - (event.pos[1] * canvas.pixel_scale)
+
+    # render a small patch around the mouse cursor
     restore_state = not face_picking_filter.enabled
-
     face_picking_filter.enabled = True
     mesh.update_gl_state(blend=False)
-    picking_render = canvas.render(bgcolor=(0, 0, 0, 0), alpha=True)
-
+    picking_render = canvas.render(
+        region=(x_pos - 1, y_pos - 1, 3, 3),
+        size=(3, 3),
+        bgcolor=(0, 0, 0, 0),
+        alpha=True,
+    )
```

Comment:
Thanks for the idea @brisvag - I think this is something like a 2-4x improvement! Is that in line with what you'd expect?

Reply:
Hm, actually using `crop` instead of `region` seems even a bit faster. This was unexpected for me but seems to agree with the docstring for `SceneCanvas._render_picking`:
>  The crop (x, y, w, h) of the framebuffer to read. For picking the full canvas is rendered and cropped on read as it is much faster than triggering transform updates across the scene with every click.

Reply:
Interesting! I definitely expected `region` to be much faster, but if there's transform updates involved I guess it makes sense. 2-4x is already pretty good though :)